src/core/src/test/java/org/locationtech/geogig/storage/ObjectStoreConformanceTest.java

```diff
@@ -77,6 +100,13 @@ public void setUp() throws IOException {
 
         this.db = createOpen(platform, hints);
         this.db.open();
+
+        dbWriteThreads = Executors.newFixedThreadPool(CONCURRENT_THREAD_COUNT,
```

Comment:
the thread pools are being created and destroyed even for all the tests that don't use them.
Idea: can we either set them up on the tests that use them, OR, probably better, extract these tests out to a new ObjectStoreConcurrencyTest suite?